docs/_documentations/debugging.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: Debugging in Codewind
+description: Debugging in Codewind
+keywords: debug, mode, applications, application, project, menu, view, attaching, attach, actions, action, run, restart in debug mode, attach debugger, debug process, debugging process
+duration: 1 minute
+permalink: debugging
+type: document
+---
+
+# Debugging in Codewind
+
+Complete the following steps to enable debugging in Codewind applications:
+
+1. In the Codewind view, right-click a project and select **Restart in Debug Mode** from the menu.
+2. Wait for the project state to switch to **Debugging**. The first time that the project switches to **Debugging**, the debugger is attached.
+3. When in debug mode, you can perform the debugging actions that are included with your IDE.
+4. To switch back to run mode, right-click your project and select **Restart in Run Mode** from the menu.
+5. If you want to return to debug mode after detaching or ending the debug process, right-click your project and select **Attach Debugger** from the menu.
```

Comment:
Swap step 4 and 5, because switching back to run mode will kick you out of debug mode since you don't want to debug anymore.